src/__tests__/commands/locations/create.test.ts

```diff
@@ -0,0 +1,108 @@
+import { jest } from '@jest/globals'
+import { FunctionLike } from 'jest-mock'
+
+import { ArgumentsCamelCase, Argv } from 'yargs'
+
+import { Location, LocationCreate } from '@smartthings/core-sdk'
+
+import { tableFieldDefinitions } from '../../../lib/command/util/locations-util.js'
+import { APICommand, APICommandFlags, apiCommand, apiCommandBuilder, apiDocsURL } from '../../../lib/command/api-command.js'
+import { inputAndOutputItem, inputAndOutputItemBuilder } from '../../../lib/command/basic-io.js'
+import { CommandArgs } from '../../../commands/locations/create.js'
+import { LocationsEndpoint, SmartThingsClient } from '@smartthings/core-sdk'
+
+
+jest.unstable_mockModule('../../../lib/command/util/locations-util.js', () => ({
+	tableFieldDefinitions,
+}))
+
+const apiCommandMock: jest.Mock<typeof apiCommand> = jest.fn()
+const apiCommandBuilderMock: jest.Mock<typeof apiCommandBuilder> = jest.fn()
+const apiDocsURLMock: jest.Mock<typeof apiDocsURL> = jest.fn()
+jest.unstable_mockModule('../../../lib/command/api-command.js', () => ({
+	apiCommand: apiCommandMock,
+	apiCommandBuilder: apiCommandBuilderMock,
+	apiDocsURL: apiDocsURLMock,
+}))
+
+const inputAndOutputItemMock: jest.Mock<typeof inputAndOutputItem> = jest.fn()
+const inputAndOutputItemBuilderMock: jest.Mock<typeof inputAndOutputItemBuilder> = jest.fn()
+jest.unstable_mockModule('../../../lib/command/basic-io.js', () => ({
+	inputAndOutputItem: inputAndOutputItemMock,
+	inputAndOutputItemBuilder: inputAndOutputItemBuilderMock,
+}))
+
+
+const { default: cmd } = await import('../../../commands/locations/create.js')
+
+
+type BuilderFunctionMock<T extends FunctionLike> = jest.Mock<T> & T
```

Comment:
For now, I've been duplicating this but I plan to put it in a utility module when I do some refactoring after getting a couple more commands with tests converted.